app/models/project.rb

```diff
@@ -1,6 +1,7 @@
 class Project < ApplicationRecord
   belongs_to :organization
   belongs_to :project_type
+  belongs_to :project_status
```

Comment:
rename this to just status

Reply:
applies to elsewhere in this commit, model name is fine as it is

Reply:
wait idgi

Reply:
Do you want me to change the model name or leave it how it is?

Reply:
keep the model name, but change the attribute name

Reply:
wait, can you do that? so you want `Project.status` instead of `Project.project_status` right? I'm thinking of just making a model method to do that? is that conventional?